Runtime/BacktraceDatabase.cs

```diff
@@ -36,7 +36,7 @@ public IBacktraceBreadcrumbs Breadcrumbs
             {
                 if (_breadcrumbs == null)
                 {
-                    if (Enable && Configuration.EnableBreadcrumbsSupport)
```

Comment:
Is `EnableBreadcrumbsSupport` used anywhere else? Can we eliminate this variable from `Configuration` now and always use `BacktraceBreadcrumbs.CanStoreBreadcrumbs`?

Reply:
`CanStoreBreadcrumbs` is a method. `EnableBreadcrumbsSupport` is a configuration flag, that is responsible for setting breadcrumbs options at all.

Reply:
That makes sense, thank you!

Reply:
Anyway, I didn't resolve the thread beacuse you were right! We should use here EnablebreadcrumbSupport flag anyway. Thanks for this catch @rqbacktrace !